include/extractor/guidance/validation_handler.hpp

```diff
@@ -46,7 +46,8 @@ class ValidationHandler final : public IntersectionHandler
                             const EdgeID via_eid,
                             Intersection intersection) const override final
     {
-        checkForSharpTurnsOntoRamps(nid, via_eid, intersection);
+        // checkForSharpTurnsOntoRamps(nid, via_eid, intersection);
```

Comment:
@daniel-j-h so for now, I'll just uncomment your check and we'll let both of them run one after the other correct?